static/docs/get-started/install.md

```diff
@@ -81,15 +81,15 @@ $ conda install -c conda-forge dvc
 
 </details>
 
-To install the most recent development version, See our
+To install the most recent development version, see our
 [development](/doc/user-guide/development) guide.
 
 ### Shell autocomplete
 
 Visit the [Shell Autocomplete](/doc/user-guide/autocomplete) guide to learn how
-to install the completion scripts for your command-line shell.
+to install the completion scripts for your command line shell.
```

Comment:
Good call. BTW it seems we have `command-line` in 4 other docs... Would you like to find them and replace? (There's another instance of this text but it's in a link URL, so that one is correct.)